src/components/timeRangePicker/TimeRangePicker.tsx

```diff
@@ -295,3 +410,44 @@ const getDateWithTime = (date: Date, time: string): Date => {
 
   return new Date(copiedDate.setHours(timeNumbers[0], timeNumbers[1], 0));
 };
+
+const getDateWithDays = (date: Date, rangeType: RANGE_TYPE): string => {
+  if (rangeType === RANGE_TYPE.WEEK) {
+    return `${format(new Date(date), dateFormat)} - ${format(
+      addWeeks(date, 1),
+      dateFormat
+    )}`;
+  }
+  if (rangeType === RANGE_TYPE.MONTH) {
+    return `${format(new Date(date), dateFormat)} - ${format(
+      addMonths(date, 1),
+      dateFormat
+    )}`;
+  }
+  if (rangeType === RANGE_TYPE.YEAR) {
+    return `${format(new Date(date), dateFormat)} - ${format(
+      addYears(date, 1),
+      dateFormat
+    )}`;
+  }
+  return format(new Date(date), dateFormat);
```

Comment:
I think it's more readable this way

```suggestion
  if (rangeType === RANGE_TYPE.YEAR) {
    return `${format(new Date(date), dateFormat)} - ${format(
      addYears(date, 1),
      dateFormat
    )}`;
  }
  
   if (rangeType === RANGE_TYPE.DAY) {
      return format(new Date(date), dateFormat);
   }
```